live/src/fix_bootconfig

```diff
@@ -13,12 +13,18 @@
 #    This script (a) adjusts the boot config, (b) runs xorriso, and (c) makes
 #    it zIPL bootable on s390x.
 #
+set -x
```

Comment:
maybe we can be even more strict and use `ex` ?

Reply:
Works with me.